tests/foreman/ui/test_contenthost.py

```diff
@@ -1489,22 +1485,21 @@ def test_content_access_after_stopped_foreman(
         organization=org.id,
     ).create()
     lce = sat.api.LifecycleEnvironment(organization=org).create()
-    with sat:  # ensure the context section only uses this satellite
-        repos_collection = RepositoryCollection(
-            distro=DISTRO_RHEL7,
-            repositories=[
-                RHELAnsibleEngineRepository(cdn=True),
-                SatelliteToolsRepository(),
-                YumRepository(url=settings.repos.yum_1.url),
-                YumRepository(url=settings.repos.yum_6.url),
-            ],
-        )
-        repos_collection.setup_content(org.id, lce.id, upload_manifest=True)
-        repos_collection.setup_virtual_machine(rhel7_contenthost, default_sat)
+    repos_collection = RepositoryCollection(
+        distro=DISTRO_RHEL7,
+        repositories=[
+            YumRepository(url=settings.repos.yum_1.url),
+        ],
+    )
+    repos_collection.setup_content(org.id, lce.id, upload_manifest=False)
+    repos_collection.setup_virtual_machine(rhel7_contenthost, sat, install_katello_agent=False)
     result = rhel7_contenthost.execute(f'yum -y install {FAKE_1_CUSTOM_PACKAGE}')
     assert result.status == 0
-    assert default_sat.execute('systemctl stop foreman').status == 0
-    result = default_sat.execute('satellite-maintain service status --only=foreman')
+    assert sat.execute('systemctl stop foreman').status == 0
+    result = sat.execute('satellite-maintain service status --only=foreman')
     assert result.status == 1
     result = rhel7_contenthost.execute(f'yum -y install {FAKE_0_CUSTOM_PACKAGE}')
     assert result.status == 0
+    assert sat.execute('systemctl start foreman').status == 0
```

Comment:
I think destructive sat is the perfect satellite for this case even though the test is running in single thread. The only reason is `due to some reason foreman service didnt start or tracebacked, this will block further test cases !`, in short `we cannot provision a perfect teardown for this case`!

Reply:
Its marked destructive, which will block it in current pipelines. destructive marked tests do need to be moved to destructive_sat, but need to be audited for their fixture use.

@JacobCallahan thoughts on merging this as-is? I'm willing to, and we're getting close to spending time on the destructive sat tests.